pandas/tests/io/formats/test_info.py

```diff
@@ -491,3 +491,19 @@ def test_info_int_columns():
         """
     )
     assert result == expected
+
+
+def test_info_from_rec():
```

Comment:
After our discussion, I fixed that issue and also added that test in test_info.py (please check lines 502-509) plus in that test case, I made correction as well suggested by @simonjayhawkins (https://github.com/pandas-dev/pandas/pull/47319#discussion_r901629823).

Furthermore, I already have initial test case included as well (lines 498-500).

I'm unsure what else is required.

On pre-commit failure, I'm beginner here and not sure what causing it to fail (when it passed for earlier commits). Any help to get this resolved would be appreciated.

Reply:
Ah sorry, missed that. Could you split that into two tests? Since technically they are testing different things.

You can run pre-commit locally and check the log. This should show the errors